src/lib/preferences.ts

```diff
@@ -112,21 +147,39 @@ const basePreferences: Preferences = {
   'markdown.previewStyle': 'default',
   'markdown.codeBlockTheme': 'material-darker',
   'markdown.includeFrontMatter': true,
+
+  // Keymap
+  'general.keymap': null,
```

Comment:
Provide an empty map if it is not available

Reply:
Added